gradle-plugin/build.gradle

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins {
+    id 'java-gradle-plugin'
```

Comment:
is it the custom to use 4 space indents in groovy?

Reply:
Maybe 2-space is majority.

Grpc-java uses 4-space https://github.com/grpc/grpc-java/blob/master/build.gradle .
Jib uses 2-space https://github.com/GoogleContainerTools/jib/blob/master/jib-gradle-plugin/build.gradle
Apache Beam uses 2-space https://github.com/apache/beam/blob/master/build.gradle .
Gax-java uses 2-space https://github.com/googleapis/gax-java/blob/master/build.gradle

Reply:
Updated with 2-space.